### PR TITLE
Add development and watch scripts

### PR DESCRIPTION
**Pull request by alicoding**

### CODE REVIEW

**Comment by vazquez** [comment]
@thisandagain - I asked Ali to help me add a watch script to make front-end reloading better.

File: package.json [hunk 1]
```diff
@@ -3,15 +3,19 @@
...
+    "start": "npm run build && npm run server",
+    "dev": "npm-run-all --parallel server watch:**",
     "build": "npm-run-all build:**",
     "build:clean": "rimraf ./build && mkdirp ./build",
-    "build:static": "ncp ./static ./build",    
+    "build:static": "ncp ./static ./build",
     "build:css": "autoless --no-watch --autoprefix \"> 5%, last 2 versions, android >= 4.0\" ./src ./build/css",
     "build:js": "webpack",
+    "watch:js": "webpack --watch",
+    "watch:css": "autoless --autoprefix \"> 5%, last 2 versions, android >= 4.0\" ./src ./build/css",
```
Comment (thisandagain) [inline package.json:14]: This is pretty non-dry with the `build:css` script above. Take a look at https://github.com/mozilla/webmaker-android/blob/develop/package.json for example of how to thin this out.

**Comment by thisandagain** [comment]
Looks great! Just one comment.

File: package.json (hunk 1 above)
Comment (alicoding) [inline package.json:14]: good call!

**Comment by alicoding** [comment]
@thisandagain done :)

**Comment by thisandagain** [comment]
Awesome! Thanks a ton @alicoding R+

**Pull request merged**